src/components/project/DetailButton/index.js

```diff
@@ -5,17 +5,18 @@ import CloudinaryImage from '../../CloudinaryImage';
 
 import s from './styles.module.scss';
 
-const ProjectDetailButton = ({ text, className, href, ...rest }) => (
-  <a className={classnames(s.btn, className)} href={href} {...rest}>
+const ProjectDetailButton = ({ text, className, href, child = false, ...rest }) => (
+  <a className={classnames(child ? s.btnChild : s.btn, className)} href={href} {...rest}>
     {text}
-    <CloudinaryImage filename="github.svg" filters={[]} className={s.github} alt="GitHub" />
-    <CloudinaryImage filename="arrow.svg" filters={[]} className={s.arrow} alt={text} />
+    <CloudinaryImage filename="github.svg" filters={[]} className={classnames(child ? s.githubChild : s.github)} alt="GitHub" />
+    <CloudinaryImage filename="arrow.svg" filters={[]} className={classnames(child ? s.arrowChild : s.arrow)} alt={text} />
```

Comment:
I was thinking in something like:
```suggestion
    <CloudinaryImage
      filename="github.svg"
      filters={[]}
      className={classnames({
        [s.githubChild]: child,
        [s.github]: !child,
      })}
      alt="GitHub"
    />
    <CloudinaryImage
      filename="arrow.svg"
      filters={[]}
      className={classnames({
        [s.arrowChild]: child,
        [s.arrow]: !child,
      })}
      alt={text}
    />
```